js/common.js

```diff
@@ -1137,7 +1137,11 @@ $(document).ready(function() {
         }
     });
 
-    $(document).on('submit', 'form', function() {
+    $(document).on('submit', 'form', (e) => {
+        // if the submitter has a data-block-on-unsaved attribute, do not clear the unsaved changes flag
+        if ($(e.originalEvent.submitter).attr('data-block-on-unsaved') === 'true') {
+            return;
+        }
         window.glpiUnsavedFormChanges = false;
     });
 });
```

Comment:
When clicking on the `Get email tickets now` button, if the "Cancel" button is used when the "Are you sure ...." message is displayed, then the form buttons are not usable anymore:

![image](https://github.com/glpi-project/glpi/assets/33253653/1dac0f28-228e-469c-9092-e49a105ad658)

Also, I am not sure it is the best place for this kind of button. There are already some items that have action buttons in the form header, like users. Maybe we should put the button at this place. @orthagh What is your opinion?
![image](https://github.com/glpi-project/glpi/assets/33253653/ec8a9870-e7ba-4bfa-8036-46fca807ddcd)

Reply:
For one action, I prefer the current iteration (Main form I guess).
Actions in user's header is, at least for me, an aberation : exception on ui, only user has this kind of button. But now, it's difficult to move to another design.

Another design I don't like, is the global action button still in header where it should be placed side of Save button (like in tickets)

Reply:
> When clicking on the `Get email tickets now` button, if the "Cancel" button is used when the "Are you sure ...." message is displayed, then the form buttons are not usable anymore:
> 
> ![image](https://user-images.githubusercontent.com/33253653/273511919-1dac0f28-228e-469c-9092-e49a105ad658.png)

This issue is still present.